### PR TITLE
Database setup

### PR DESCRIPTION
**Pull request by vpatel90**
Added database models and resources

### CODE REVIEW

**Comment by jah2488** [comment]
Remove `.coffee` files please.

File: db/migrate/20160421205101_create_events.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class CreateEvents < ActiveRecord::Migration
+  def change
+    create_table :events do |t|
```
Comment (jah2488) [inline db/migrate/20160421205101_create_events.rb:3]: Events should belong to user

File: db/migrate/20160421205419_create_amenities.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class CreateAmenities < ActiveRecord::Migration
+  def change
+    create_table :amenities do |t|
```
Comment (jah2488) [inline db/migrate/20160421205419_create_amenities.rb:3]: I believe amenities need a name.

File: db/migrate/20160421205425_create_comments.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
```
Comment (jah2488) [inline db/migrate/20160421205425_create_comments.rb:3]: I think comments need to have a text field for the comment itself.

**Comment by jah2488** [comment]
👍

**Comment by reckocp** [comment]
👍

**Comment by reicht** [comment]
👍

**Pull request merged**